### ISSUE TITLE
Parameters in the execution in Java

### ISSUE THREAD

**Issue by miatauro-NASA** [nasa/europa#141]
_From [None](https://code.google.comNone) on September 13, 2011 05:04:03_

_Original issue: http://code.google.com/p/europa-pso/issues/detail?id=141_

**Comment by CharlyDelta** [comment]
Is there an answer for this issue? I have the same problem and I dont find anything in the documentations which could help me

**Comment by miatauro-NASA** [comment]
The immediate answer to this question is:
1) Either "g" or "o" to indicate that it should load the debug or optimized library, respectively.
2) The name of the .bsh file to load (all the real work is done in a BeanShell script).
3)  Optionally "nogui" to avoid loading any UI.  The planner will run on the console.

The medium-term answer is that this needs to be documented on the wiki.

The longer-term answer is that this needs to be self-documenting.  Tickets will be forthcoming!

**Comment by miatauro-NASA** [comment]
Christian,

My apologies, both for the longevity of this question and that it needed asking at all. 
I've added a comment to issue 141 with an answer:

1) Either "g" or "o" to indicate that it should run with a debug or optimized library, 
respectively.
2) The name of the BeanShell script for it to launch (most of the loading work is done in 
these scripts).
3) Optionally "nogui" to run the planner in the console without any graphical interface.

~MJI

On 04/22/2015 05:48 AM, Christian Dreher wrote:
> Is there an answer for this issue? I have the same problem and I dont find anything in the
> documentations which could help me
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nasa/europa/issues/141#issuecomment-95160859>.
>


-- 
------------------
Michael J. Iatauro
Software Engineer, IRIS Flight Controller
QTS, Inc.

NASA Ames Research Center
Office: 650-604-0662
Mail stop: 269-2
P.O. Box 1
Moffett Field, CA 94035-0001

**Comment by CharlyDelta** [comment]
Thank you very much! I eventually used ANT, which seems to be working better though

**Comment by miatauro-NASA** [comment]
By "used ANT", do you mean that you used ant to run the Light example?

~MJI

On 04/22/2015 08:28 AM, Christian Dreher wrote:
> Thank you very much! I eventually used ANT, which seems to be working better though
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nasa/europa/issues/141#issuecomment-95236965>.
>


-- 
------------------
Michael J. Iatauro
Software Engineer
QTS, Inc.

NASA Ames Research Center
Office: 650-604-0662
Mail stop: 269-2
P.O. Box 1
Moffett Field, CA 94035-0001

**Comment by CharlyDelta** [comment]
Yes exactly. Why? Would that turn out to be a problem as soon as I make own projects?

**Comment by miatauro-NASA** [comment]
On 04/22/2015 12:16 PM, Christian Dreher wrote:
> Yes exactly. Why? Would that turn out to be a problem as soon as I make own projects?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nasa/europa/issues/141#issuecomment-95308479>.
>
No, I just wanted to make sure I understood how you worked around the problem.  The 
project design doesn't incorporate easily adding command-line arguments, so using ant to 
start things won't make it any more difficult than doing so manually.

~MJI

-- 
------------------
Michael J. Iatauro
Software Engineer
QTS, Inc.

NASA Ames Research Center
Office: 650-604-0662
Mail stop: 269-2
P.O. Box 1
Moffett Field, CA 94035-0001

**Comment by CharlyDelta** [comment]
Ah all right, great. :) I think I will try to use ANT because it seems quite comfortable. 

Thank you very much for your quick responses and help, I really appreciate that!
